# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/sparc-weak-asm.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./sparc/strcmp.S.orig	2004-09-19 14:32:02.524790856 +0200
+++ ./sparc/strcmp.S	2004-09-19 14:32:29.443698560 +0200
@@ -11,6 +11,8 @@
 
 .text
 .globl strcmp
+.weak strcoll
+strcoll:
 strcmp:
 	clr %o4
 .Lloop:
--- ./sparc/strchr.S.orig	2004-09-19 14:30:12.402531984 +0200
+++ ./sparc/strchr.S	2004-09-19 14:31:56.704675648 +0200
@@ -11,6 +11,8 @@
 
 .text
 .globl strchr
+.weak index
+index:
 strchr:
 	ldub [%o0], %o2
 	cmp %o2, %o1
